docs/components/modeler/web-modeler/advanced-modeling/form-linking.md

```diff
@@ -48,6 +52,35 @@ To deploy to a Camunda 8 cluster with a version lower than 8.4, linked forms wil
 This conversion will only be applied to the XML deployed to the cluster; the diagram in Web Modeler will not be changed.
 :::
 
+#### Known issues with linked forms
+
+##### Wrong form used for user task - [zeebe/#16311](https://github.com/camunda/zeebe/issues/16311)
+
+Some users have encountered an issue that was present in Camunda `8.4.0`, `8.4.1`, and `8.4.2` where linked forms did not correspond correctly to their tasks, leading to discrepancies in workflow execution. We have identified and rectified this issue in the Camunda `8.4.3` release, ensuring that tasks are now generated with the correct forms, as intended in your workflow design.
+
+###### How to fix the issue - Regenerate tasks with correct forms
+
+To correct any instances affected by this issue, we recommend the following steps after updating to Camunda `8.4.3`:
+
+1. Navigate to [Operate](/components/operate/operate-introduction.md) to access your workflow instances impacted by the linked form issue.
+2. Locate the instances where tasks were generated with the incorrect forms.
+3. Move the instance to the same task, effectively restarting the task. To do this, click the wrench icon in the top right corner:
+
+<img src={IssueLinkedFormSolution01} style={{width: 400}} alt="Modify Process Instance on Operate" />
+
+4. After this, a popup explaining how process modification works will appear. Click **Continue**, select the active task, and click **Cancel instance**.
+
+<img src={IssueLinkedFormSolution02} style={{width: 400}} alt="process instance modification mode" />
```

Comment:
I moved these images around based on the referenced steps and adjusted the alt text. Does this look correct?

Reply:
Sorry, It was not clear for me this @christinaausley